SPECS/numpy/numpy.spec

```
@@ -63,6 +66,9 @@ PYTHONPATH=%{buildroot}%{python3_sitelib} PATH=$PATH:%{buildroot}%{_bindir} %pyt
 %{_bindir}/f2py%{python3_version}
 
 %changelog
+* Thu Sep 15 2022 Riken Maharjan <rmaharjan@microsoft.com> - 1.22.3-2
+- replaced py3_install macro to include numpy header files. --skip-build arg is removed essentially with the py3_install macro removal.
```

NIt-pick: please start changelog entries from capital letters to be consistent with what we've been doing elsewhere.

```suggestion
- Replaced py3_install macro to include numpy header files. --skip-build arg is removed essentially with the py3_install macro removal.
```